### PR TITLE
feat(unlock-app) - add download csv

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description
**Download CSV Button**

Screenshot 2022-10-05 at 19 39 18

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by kalidiagne** [comment]
> IIRC we had 2 version of the CSV export: the full one (which we should absolutely have!) and a partial one (which we can get rid of), based on what is "shown" on the screen (so taking the filtering + pagination) into account.
> 
> Which one is this?

This is the full one!

**Review by julien51 — APPROVED** [review]

File: unlock-app/src/components/interface/locks/Manage/index.tsx [hunk 1]
```diff
@@ -20,21 +20,70 @@ import { useConfig } from '~/utils/withConfig'
...
+        query: '',
+        filterKey: 'owner',
+        expiration: 'all',
+      },
+    })
+  }
+
+  const onDownloadCsv = async () => {
+    const members = await getMembers()
+    downloadAsCSV(defaultCols, members)
```
Comment (julien51) [inline unlock-app/src/components/interface/locks/Manage/index.tsx:85]: Wait we need the user to be able to download _all_ metadata, not just the ones in `defaultCols`.

**Review by julien51 — CHANGES_REQUESTED** [review]

File: unlock-app/src/components/interface/locks/Manage/index.tsx (hunk 1 above)
Comment (kalidiagne) [reply unlock-app/src/components/interface/locks/Manage/index.tsx]: fixed, cols are now taken from members list

**Comment by kalidiagne** [comment]
**CSV Dowloading**
Screenshot 2022-10-07 at 10 15 18

**CSV Downloaded** 
Screenshot 2022-10-07 at 10 16 09

**Review by julien51 — APPROVED** [review]

**Pull request merged**